.github/workflows/sass.yaml

```
@@ -1,94 +1,115 @@
 name: Sass
 
-on: [push, pull_request]
+on:
+  pull_request:
+
+  push:
+    branches:
+      - main
+      - 'feature/**'
+      - 'support/**'
+
+  workflow_dispatch:
+
+concurrency:
+   group: sass-${{ github.ref_name }}
```

Does a fallback for `ref_name` need to be used for pushes without a pull request? [^1]

[^1]: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value

---

Hi @injms, thanks for watching out for us 😊 

Did you see I'd used `${{ github.ref_name }}` **refs/heads/main** instead of `${{ github.head_ref }}`?

Appreciate how the docs would give a nicer result though, happy to change

```suggestion
   group: sass-${{ github.head_ref || github.ref_name }}
```

---

So many similarly-named-variables! Apologies, I didn't spot that you'd used `ref_name`, so clearly no need to change it 👍

---

To be fair, was a good spot!

Two merges into `main` would share the same `${{ github.ref_name }}` so the 2nd push would cancel the 1st

I've used [your (and GitHub's) suggestion](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value) to fall back to the (always-unique) `github.run_id` outside PRs